lib/src/app/feature/google_authenticaiton/bloc/authentication_state.dart

```diff
@@ -0,0 +1,30 @@
+part of 'authentication_bloc.dart';
+
+abstract class AuthenticationState extends Equatable {
+  const AuthenticationState();
+
+  @override
+  List<Object> get props => [];
+}
+
+class AuthenticationInitial extends AuthenticationState {}
+
+class AuthenticationLoading extends AuthenticationState {}
+
+class AuthenticationFailiure extends AuthenticationState {
+  final String message;
+  AuthenticationFailiure({
```

Comment:
Can you please move this constructor first?
https://dart-lang.github.io/linter/lints/sort_constructors_first.html